tests/integration/conftest.py

```diff
@@ -11,26 +11,6 @@
 from tenacity import Retrying, stop_after_delay, wait_fixed
 
 
-@pytest.fixture(scope="module")
-def ops_test(
-    ops_test: pytest_operator.plugin.OpsTest, pytestconfig
-) -> pytest_operator.plugin.OpsTest:
-    _build_charm = ops_test.build_charm
-
-    async def build_charm(charm_path) -> pathlib.Path:
-        if pathlib.Path(charm_path) == pathlib.Path("."):
-            # Building sysbench charm
-            return await _build_charm(
-                charm_path,
-                bases_index=pytestconfig.option.sysbench_charm_bases_index,
-            )
-        else:
-            return await _build_charm(charm_path)
-
-    ops_test.build_charm = build_charm
-    return ops_test
```

Comment:
This is no longer needed by the reusable workflow and hides the extra params of the original `build_charm`